website/docs/shaderobjectfileoutput.md

```diff
@@ -18,7 +18,9 @@ Premake 5.0.0 alpha 14 or later.
 
 ### Examples ###
 
+This project will compile HLSL files to the shaders folder with a .cso extension.
+
 ```lua
-shaderobjectfileoutput (value)
+shaderobjectfileoutput "shaders/%%(Filename).cso"
```

Comment:
Nit: Choose a slightly simpler example that doesn't involve escaping.

Reply:
For Visual Studio projects at least, this path will be dynamic based on the item being compiled.  Without the use of some form of macro all shader files would be compiled with the same file name.  I haven't used premake for other project types but I would assume this would also be the case for projects other than Visual Studio.  I've updated the example to make it clear that this would work specifically for a Visual Studio project which should be sufficient for most users?